Services/CO.CDP.OrganisationInformation.Persistence/DatabaseOrganisationRepository.cs

```diff
@@ -271,8 +271,7 @@ public async Task<int> GetTotalCount(PartyRole? role, PartyRole? pendingRole, st
 
         if (searchText != null)
         {
-            result = result.Where(o => o.Name.Contains(searchText) ||
-                                       EF.Functions.TrigramsSimilarity(o.Name, searchText) > 0.3);
+            result = result.Where(o => o.Name.ToLower().Contains(searchText.ToLower()));
```

Comment:
I think this logic still doesn't sufficiently match what's in `GetPaginated` ? So the count will not match the number of results.